modules/aws-outposts-prereq-storage-csi.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-prereq-storage-csi_{context}"]
+= Preparing the AWS EBS CSI driver to support AWS Outposts
+
+//to-do: it would be better to employ reuse here so that these steps can't become outdated if something changes in the storage procedures. Need to make snippets from the following modules (in consultation with Storage writer):
+//* modules/persistent-storage-csi-sc-managing-cli.adoc (change to Unmanaged)
+//* modules/dynamic-provisioning-change-default-class.adoc (change default SC)
+
+You must update the default storage class for the AWS Elastic Block Store (EBS) Container Storage Interface (CSI) driver to be compatible with AWS Outposts.
```

Comment:
Since we `Specifies a taint to prevent user workloads from being scheduled on nodes that have the `node-role.kubernetes.io/outposts` label.`,  from my understanding, change the default storageclass to `gp2-csi` become an optional step. If user doesn't change the default storageclass, deploy application with persistent volume on outposts nodes should specified the `gp2-csi` storageclass.
```suggestion
Better to update the default storage class for the AWS Elastic Block Store (EBS) Container Storage Interface (CSI) driver to be compatible with AWS Outposts.
```

Reply:
yes - I was asked to remove this topic in favor of the approach using taints. I had kept it in the PR for reference but i will pull the info that is still valid from it and remove it to prevent confusion.